docs/orchestration/index.md

```diff
@@ -233,103 +169,26 @@ stack {
 }
 ```
 
-The order of execution will be:
+With these settings, the execution sequence will be **stack-a** -> **stack-c** -> **stack-b**.
 
-* stack-a
-* stack-c
-* stack-b
-
-One example of terramate command that leverages order of
-execution is **terramate run**.
-
-This will run **terraform** plan on all stacks, but respecting ordering:
+One command that considers this execution order is `terramate run`, which runs `terraform plan` on all stacks in the defined sequence:
```

Comment:
this phrase is misleading as it imply that `terramate run` automatically runs `terraform plan` on all stacks..